### PR TITLE
[DSK-25] Project setup

### PR DESCRIPTION
**Pull request by jan-zabloudil**

### CODE REVIEW

File: supabase/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Supabase
```
[the remark this answers is not in the document]
Comment (jan-zabloudil) [reply supabase/.gitignore:1]: `supabase/.gitgnore` is automatically generated when you run `supabase init`. I guess this .env is really related only to Supabase.

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"errors"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+type Config struct {
```
[the remark this answers is not in the document]
Comment (jan-zabloudil) [reply config/config.go]: I wasn't sure what is the best approach when using go.strv.io/env for following case 

```
SecretKey string `env:"SUPABASE_SECRET_KEY"`
```

Let's say SUPABASE_SECRET_KEY is **required env variable**.

**Options**

1. Define custom type for secret key
```
type secretkey string 

func (s *supabaseSecretKey) UnmarshalText(text []byte) error {
	if *s == "" {
		panic("...")
	}
	return nil
}
```

2. Just write check after `envx.MustApply(&cfg)`

```
if cfg.Supabase.SecretKey == "" {
   panic("..")
} 
```

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+* @CermakM @jan-zabloudil @bolshakoff
```
Comment (jan-zabloudil) [inline CODEOWNERS:1]: @bolshakoff I've invited you to be a collaborator, once you accept, error is fixed

Comment (bolshakoff) [reply CODEOWNERS:1]: Accepted the invitation. 🙏

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+4. To test if Google OAuth was set up correctly, open your browser and navigate to `<SUPABASE_API_URL>/auth/v1/authorize?provider=google`.
+
+### REST API
+
+1. Now set env variables for REST API:
+
+| Env variable                          | Where do I find this value?                                                                                                                                                           | Default |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `RELEASE_MANAGER_PORT`                | -                                                                                                                                                                                     | `8080`  |
+| `RELEASE_MANAGER_LOG_LEVEL`           | Allowed values `DEBUG`, `INFO`, `WARN`, `ERROR`.                                                                                                                                      | `DEBUG` |
```
Comment (bolshakoff) [inline README.md:32]: I would expect at least `INFO` as the default log level.

Comment (jan-zabloudil) [reply README.md]: Default is now `INFO`

**Pull request merged**